### PR TITLE
Fix for 1298

### PR DESCRIPTION
**Pull request by sgpearse**
Fix for #1298 (slice not agreeing with volume renderer region)

### CODE REVIEW

File: apps/vaporgui/CopyRegionWidget.cpp [hunk 1]
```diff
@@ -110,6 +114,29 @@ void CopyRegionWidget::updateCopyCombo() {
...
+        int orientation = myBox->GetOrientation();
+        if (orientation == Box::XY)
+            planarAxis = Z;
+        else if (orientation == Box::XZ)
+            planarAxis = Y;
+        else if (orientation == Box::YZ)
+            planarAxis = X;
+        else 
+            return;
+
```
[the remark this answers is not in the document]
Comment (sgpearse) [reply apps/vaporgui/CopyRegionWidget.cpp]: @clyne, ok, it should be good to go now.

[the remark this answers is not in the document]
Comment (clyne) [reply apps/vaporgui/CopyRegionWidget.cpp]: The code is unchanged.

File: include/vapor/Box.h [hunk 2]
```diff
@@ -43,6 +43,7 @@ class PARAMS_API Box : public ParamsBase {
         XY = 0,
         XZ = 1,
         YZ = 2,
+        THREE_D = 3
```
[the remark this answers is not in the document]
Comment (sgpearse) [reply include/vapor/Box.h]: I see what you're saying.  I think the problem is bigger than what's involved with this bug though.  I'll try to rename things to make more sense, but a lot of our legacy code needs to be reassessed from a high level, now that our requirements are easier for us to grasp at this point in development.

File: apps/vaporgui/CopyRegionWidget.h [hunk 3]
```diff
@@ -39,6 +39,11 @@ private slots:
 
 private:
 	void updateCopyCombo();
+    void _configurePlanarBox(
```
[the remark this answers is not in the document]
Comment (sgpearse) [reply apps/vaporgui/CopyRegionWidget.h]: No problem.

File: apps/vaporgui/CopyRegionWidget.cpp [hunk 4]
```diff
@@ -143,7 +170,10 @@ void CopyRegionWidget::copyRegion()
 			myMin[i] = minExtents[i];
 			myMax[i] = maxExtents[i];
 		}
-		myBox->SetExtents( myMin, myMax );
+      
+        _configurePlanarBox( myBox, &myMin, &myMax);
```
[the remark this answers is not in the document]
Comment (sgpearse) [reply apps/vaporgui/CopyRegionWidget.cpp]: The function returns!

**Pull request merged**